Author: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>

Fix FTBFS on Renesas SH (sh4) because libjpeg doesn't link against sh772x driver.
See http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=536314

--- a/gfxdrivers/sh772x/Makefile.am
+++ b/gfxdrivers/sh772x/Makefile.am
@@ -32,6 +32,8 @@ libsh7722_jpeg_la_LDFLAGS = \
 	-export-dynamic		\
 	-avoid-version
 
+libsh7722_jpeg_la_LIBADD = \
+        $(LIBJPEG)
 
 bin_PROGRAMS = sh7722_jpegtool
 
